pkg/loader/load.go

```diff
@@ -211,6 +214,10 @@ func (s *loaderImpl) refreshTableInfo(schema string, table string) (info *tableI
 		return info, errors.Trace(err)
 	}
 
+	if len(info.columns) == 0 {
```

Comment:
It would be more clear here if we have a comment to explain why empty `info.columns` means and only means "table not exist".